### PR TITLE
fix: vertically center checkbox in grid cell

### PR DESCRIPTION
**Pull request by inikolova**

### CODE REVIEW

File: packages/bootstrap/scss/grid/_variables.scss [hunk 1]
```diff
@@ -24,6 +24,8 @@ $grid-cell-padding-y: $grid-padding-y !default;
 $grid-cell-horizontal-border-width: 0px !default;
 $grid-cell-vertical-border-width: 1px !default;
 
+$grid-cell-checkbox-margin-y: calc( #{$cell-padding-y} - (#{$button-calc-size} - #{$line-height-em}) / 2 ) / 1.25  !default; // $grid-command-cell-padding-y / 1.25
```
Comment (PreslavKozovski) [inline packages/bootstrap/scss/grid/_variables.scss:27]: This is not a valid value. Here, we are assuming that the `postcss-calc` will remove the `calc`, however, if any of these variables change, and the `calc` is not removed, we will have an invalid output of the CSS.

**Review by PreslavKozovski — CHANGES_REQUESTED** [review]
The below comment is applicable to most of the `$grid-cell-checkbox-margin-y` variables.

**Review by PreslavKozovski — APPROVED** [review]

**Comment by joneff** [comment]
Due to upcoming changes in the buttons, this fix will not be relevant as it depends on variables related to the current state of the button. Converting to draft for the time being

**Comment by joneff** [comment]
To align checkboxes correctly, we need a wrapper around them (`k-checkbox-wrap`). Then and only then we can align them correctly, be it top or center align.

**Pull request closed without merging**